articles/overview/what-is-qsharp.md

```diff
@@ -1,50 +1,70 @@
 ---
-title: What is Q#?
-description: Learn about the Q#, a programming language created by Microsoft to develop applications for quantum computers
+title: What are Q# and the QDK?
+description: Learn about Q#, a programming language created by Microsoft to develop applications for quantum computers, and a key component of Microsoft's Quantum Development Kit
 author: natke
 ms.author:  nakersha
 ms.date: 10/22/2019
 ms.topic: article
 uid: microsoft.quantum.overview.qsharp
 ---
 
-# What is Q#?
+# What are Q# and the QDK?
 
-Q# is a programming language with features that are special to quantum computing.
+Q# is a programming language with features specifically designed for use with quantum computing.
+A key component of Microsoft's Quantum Development Kit (QDK), it provides quantum programmers a framework that allows you to focus on the algorithms without having to worry about technical details like gate sequence optimization or the physical implementation of a quantum computer.
 
-Q# provides quantum programmers a framework that allows you to focus on the algorithms without having to care about technical details like gate sequence optimization or the physical implementation of a quantum computer.
+The QDK comprises a wide range of tools which give developers everything they need to start writing quantum programs.
+Alongside the Q# language, the QDK includes:
+* the *Q# libraries*, which allow developers to hit the ground running and create real-world quantum applications today
+* various *target machines* on which Q# programs can be run. These include resource estimators and simulators for larger quantum programs, as well as a full-state quantum simulator, which behaves as a noise-free quantum computer. The latter is very useful for tinkering with ideas, debugging programs, and learning about quantum physics, but only efficient for programs with relatively few qubits. We're very much looking forward to introducing real quantum computers as target machines in the future.
```

Comment:
I think the last sentence can be "We're very much looking forward to introducing quantum computing hardware..." since "real" is not quite appropriate and we are introducing hardware with IonQ and Honeywell.